### PR TITLE
Add section for unraid auto-restarts

### PR DESCRIPTION
**Pull request by asherbig**
This is a mitigation related to [#731](https://github.com/cross-seed/cross-seed/issues/731). Since that issue can be handled by properly configuring auto-restarts, I thought it would be nice to explicitly add it to the documentation (along with warning users that there is a need to configure their container like this). 

The other daemon "installation" methods include auto-restarts, so this should be bringing the Unraid install docs up to parity with the others. 
* https://www.cross-seed.org/docs/basics/daemon#docker
* https://www.cross-seed.org/docs/basics/daemon#systemd-linux

Note: I personally don't see this as a fix for [#731](https://github.com/cross-seed/cross-seed/issues/731), because I think a better user experience would be for transient network issues to auto-retry by default. But, I don't want to discuss what the "correct" solution is for #731 in this PR, since it is just a docs improvement.

EDIT: here is what the page looks like locally
image

### CODE REVIEW

File: docs/tutorials/unraid.md [hunk 1]
```diff
@@ -90,3 +90,19 @@ If you would like to schedule a periodic scan of your library, use the [`searchC
 If you would like to set up RSS scans, use the [`rssCadence`](../basics/options.md#rsscadence) option.
 
 If you would like to automatically check for cross-seeds when a download finishes, learn how to [set up with automatic searches](../basics/daemon#set-up-automatic-searches-for-finished-downloads).
+
+## Auto-Restart
+
+`cross-seed` by default will stop the container if a configuration option is suspected to be bad. This includes bad config options for the torrent client.
```
Comment (bakerboy448) [inline docs/tutorials/unraid.md:96]: Adding this blurb is all that makes sense.

Comment (asherbig) [reply docs/tutorials/unraid.md:96]: If a user reads this, then their next question is going to be "why does that matter to me", and if they know why that matters to them, then it'll be "how do I prevent those problems from happening". I think the docs added address those questions in a user-friendly way.

File: docs/tutorials/unraid.md [hunk 2]
```diff
@@ -90,3 +90,19 @@ If you would like to schedule a periodic scan of your library, use the [`searchC
 If you would like to set up RSS scans, use the [`rssCadence`](../basics/options.md#rsscadence) option.
 
 If you would like to automatically check for cross-seeds when a download finishes, learn how to [set up with automatic searches](../basics/daemon#set-up-automatic-searches-for-finished-downloads).
+
+## Auto-Restart
```
Comment (zakkarry) [inline docs/tutorials/unraid.md:94]: Belongs under configuration - not as a main heading

File: docs/tutorials/unraid.md (hunk 1 above)
Comment (zakkarry) [inline docs/tutorials/unraid.md:96]:
```suggestion
`cross-seed`, by default, will stop the container if a configuration option is suspected to be bad or transient network failures occur during startup validation (such as failing to connect to the torrent client or Torznab endpoints).

File: docs/tutorials/unraid.md [hunk 3]
```diff
@@ -90,3 +90,19 @@ If you would like to schedule a periodic scan of your library, use the [`searchC
...
+
+`cross-seed` by default will stop the container if a configuration option is suspected to be bad. This includes bad config options for the torrent client.
+
+This can lead to issues where `cross-seed` may fail to start because of a temporary issue with the torrent client, like long start time, network issues, etc.
+After this happens, the container will stay **stopped** until you manually start it again, or another scheduled process turns it back on.
+
+Because of this, it is recommended to add the auto-restart parameter to your `Extra Parameters` section in Unraid.
```
Comment (zakkarry) [inline docs/tutorials/unraid.md:101]:
```suggestion
If you have verified your settings after your initial setup and are only encountering errors intermittently, you can utilize the `Extra Parameters` section of the `Advanced Settings` in the container template to automatically restart the container.

File: docs/tutorials/unraid.md [hunk 4]
```diff
@@ -90,3 +90,19 @@ If you would like to schedule a periodic scan of your library, use the [`searchC
...
+
+![screenshot-cross-seed-extra-param-config](../../static/img/unraid-extra-parameters-restart.png)
+
+:::tip
+You need to toggle from `Basic View` to `Advanced View` to see this option
+![screenshot-cross-seed-extra-param-config](../../static/img/toggle-advanced-view-unraid.png)
+:::
```
Comment (zakkarry) [inline docs/tutorials/unraid.md:108]: The tip is not needed as it is mentioned in the previous suggestion, and if you look at the other images included in the unraid tutorial you will see they are hosted as an asset on gh's cdn.

Simply paste the image into a comment box here on the cross-seed repo and it will upload it, and you can grab the link there.

**Review by zakkarry — CHANGES_REQUESTED** [review]
As baker said, this is a bit wordy for what it seeks to do, I've made some suggestions.

We're also discussing adding this to the unraid template for v6.

**Comment by asherbig** [comment]
Thanks for the review. Changes have been made!

**Review by zakkarry — APPROVED** [review]

**Comment by zakkarry** [comment]
I've moved it to the top of the configuration section since its at the top of the options, just seemed logical to put that there i guess. I dunno.

Minor language changes after re-reading on a new day.
